examples/transit_vnet_common/files/init-cfg.sample.txt

```diff
@@ -1,9 +1,8 @@
 type=dhcp-client
-vm-auth-key=123456789012345
```

Comment:
I updated documentation to point source of truth. According to documentation for 9.1 version our init-cfg is wrong. Maybe because it has not been tested enough ;), please approve.

Reply:
I've re-reviewed and I still do not see how is this change is improving anything. The `*.sample.*` file is named so to be illustrative for users. It is not intended to be used for any testing scenario - maybe we need a different file for a testing scenario, I do not have a problem with the idea.

If user sees only this:
```
vm-auth-key=
```
They may put:
```
vm-auth-key=I11223344
```
Or maybe

```
vm-auth-key="I11223344"
```

Or maybe
```
vm-auth-key='111111111111111111111234'
```

These are *all* wrong ideas, despite being plausible. If the same user sees this:

```
vm-auth-key=123456789012345
```
They will probably understand what can they put here, because they have a full example.

By *user* I mean me, next month.

If you claim that the docs page is more readable, yes, we could use as well this:

```
vm-auth-key=7550362253****
```